docs/content/configuration-language.mdx

```diff
@@ -511,6 +513,76 @@ This particular use of referencing relations on related objects is defining a tr
 
 This can be used to indicate that **viewers** of a **folders** are **viewers** of all **documents** in that **folder**.
 
+:::caution
+<details>
+<summary>
+  Note that <ProductName format={ProductNameFormat.LongForm}/> does not evaluate the related object (the word after `from`, also called the tupleset), it will only consider directly related objects, and will not perform any evaluation (e.g. any rewrites on that relation, or `*`, etc..).
```

Comment:
```suggestion
  Note that <ProductName format={ProductNameFormat.LongForm}/> does not evaluate the related object (the word after `from`, also called the tupleset). Instead, it will only consider directly related objects and will not perform any evaluation (e.g. any rewrites on that relation, or `*`, etc..).
```

I am still a bit unsure about using the term "directly related object" as it implies direct relationship.  We explained that direct relationship includes * (https://openfga.dev/docs/configuration-language#the-direct-relationship-keyword).  Maybe use the term directly assigned objects?

Reply:
```suggestion
  Note that <ProductName format={ProductNameFormat.LongForm}/>, when searching tuples that are related to the object (the word after `from`, also called the tupleset), it will only consider concrete objects that were directly assigned, and will throw an error if it encounters a`*` or a userset.
```